### ISSUE TITLE
[Android native] Libraries not worked

### ISSUE THREAD

**Issue by dmitryhryppa** [Kode/Kha#419]
Looks like Kore libraries not worked anymore with android-native target.
I think the problem is here, haxe side:

```
    public static function init(loginCallback:String->Void, sendChatMessageCallback:String->Void, viewCloseCallback:Void->Void):Void {
        #if (sys_ios || sys_android_native)
        untyped __cpp__('UrboNative::init(loginCallback, sendChatMessageCallback, viewCloseCallback);');
        #end
    }
```

and generated cpp:

```
void UrboNativeHx_obj::init( ::Dynamic loginCallback, ::Dynamic sendChatMessageCallback, ::Dynamic viewCloseCallback){
                HX_STACK_FRAME("UrboNativeHx","init",0x4897ec6d,"UrboNativeHx.init","UrboNativeHx.hx",23,0xdabb0b2d)
                HX_STACK_ARG(loginCallback,"loginCallback")
                HX_STACK_ARG(sendChatMessageCallback,"sendChatMessageCallback")
                HX_STACK_ARG(viewCloseCallback,"viewCloseCallback")
                }
```

Untyped code is gone :( **@:functionCode** and **@:headerCode** do the same effect.

**Comment by dmitryhryppa** [comment]
Problem was in sys_android_native. It's does not exist anymore: https://github.com/KTXSoftware/Kha/issues/363

**Comment by RobDangerous** [comment]
Fixed, see #363
